src/App.js

```diff
@@ -1,14 +1,243 @@
+// Import css file
 import './App.css';
+
+// Importing Firebase 
 import firebase from './firebase';
+import { getDatabase, ref, onValue, push, remove } from 'firebase/database'
+
+// Importing react Hooks
+import { useEffect, useState } from 'react';
+
+// Importing Components
+import Header from './Header';
+import Splitter from './Splitter';
 
 
 
 function App() {
-  return (
-    <div className="App">
 
-    </div>
-  );
+    // initialize our stateful variables
+    const [nameInput, setNameInput] = useState([]);
+    const [billInput, setBillInput] = useState(null);
+    const [checked, setChecked] = useState(false);
+    const [diners, setDiners] = useState({
+        nameOne: "",
+        nameTwo: "",
+        nameThree: "",
+        nameFour: "",
+        nameCheckbox: 'nameOneCheckbox',
+        bill: 0,
+    })
+
+    useEffect(() => {
+        console.log(diners);
+    }, [diners]);
```

Comment:
It looks like this console log could be taken out, unless you're still using it to debug or test something.